bluebottle/statistics/migrations/0005_auto__add_field_statistic_language__add_field_statistic_title_nl__add_.py

```diff
@@ -0,0 +1,55 @@
+# -*- coding: utf-8 -*-
+from south.utils import datetime_utils as datetime
+from south.db import db
+from south.v2 import SchemaMigration
+from django.db import models
+
+
+class Migration(SchemaMigration):
+
+    def forwards(self, orm):
+        # Adding field 'Statistic.language'
+        db.add_column(u'statistics_statistic', 'language',
+                      self.gf('django.db.models.fields.CharField')(default='', max_length=100, blank=True),
+                      keep_default=False)
+
+        # Adding field 'Statistic.title_nl'
+        db.add_column(u'statistics_statistic', 'title_nl',
+                      self.gf('django.db.models.fields.CharField')(max_length=100, null=True, blank=True),
+                      keep_default=False)
+
+        # Adding field 'Statistic.title_en'
+        db.add_column(u'statistics_statistic', 'title_en',
+                      self.gf('django.db.models.fields.CharField')(max_length=100, null=True, blank=True),
+                      keep_default=False)
+
```

Comment:
Would it be hard to create a version of  ModelTranslation that looks at the properties instead of the settings?

One thing we could do is change this migration to use the actual properties and a correct columns accordingly. Changing a sides language will still fail however.